ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__slcm-4.12.yaml

```diff
@@ -23,7 +23,7 @@ tests:
       OCP_VERSION: "4.12"
       RUN_EDU_TESTS: "true"
       SITE_NAME: cloudransno-site3
-    workflow: telcov10n-slcm
+    workflow: telcov10n-slcm-slcm-workflow
```

Comment:
I suggest we remove "workflow" from the name since the stanza already implies that it's a workflow. 
Additionally, we can remove one slcm occurrence from the workflow name. How about something like telcov10n-slcm-lab or a similar option?

Reply:
This is the old workflow with GitLab. I didn't want to change things here as part of this PR since we are going to remove it eventually and this PR already looks big

Reply:
Renamed steps